tiny_dnn/util/nms.h

```diff
@@ -0,0 +1,69 @@
+/*
+  Copyright (c) 2013, Taiga Nomi and the respective contributors
+  All rights reserved.
+
+  Use of this source code is governed by a BSD-style license that can be found
+  in the LICENSE file.
+*/
+#pragma once
+
+#include <vector>
+
+namespace tiny_dnn {
+
+struct bounding_box {
+  float x_min;
+  float y_min;
+  float x_max;
+  float y_max;
+  float score;
+
+  inline float area() const { return (x_max - x_min) * (y_max - y_min); }
+};
+
+float iou(const bounding_box &b1, const bounding_box &b2) {
```

Comment:
Since tiny-dnn is header-only library, all the non-template functions in header files have to be `static` or `inline`.